react/src/input.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
I don't think these need to be added to three files but the config docs are unclear and I haven't narrowed down which it is yet, so to-do

Reply:
It turns out it is `App.css`. I'm not sure if I even need `input.css`, will look into it

Reply:
Yeah, I think this is just an issue with Step 4 of the Tailwind docs: https://tailwindcss.com/docs/installation. Looks like that file's not necessary, the library is working without it

Reply:
Removed, marking resolved